qatrfm/environment.py

```diff
@@ -58,8 +58,7 @@ def __init__(self, image, tf_file=None, num_domains=1,
         self.snapshots = snapshots
         letters = string.ascii_lowercase
         self.basename = ''.join(random.choice(letters) for i in range(10))
-        self.workdir = self.BASEDIR + self.basename
-        os.makedirs(self.workdir)
+        self.workdir = tempfile.mkdtemp()
         self.logger.debug("Using working directory {}".format(self.workdir))
         shutil.copy(self.tf_file, self.workdir + '/env.tf')
         os.chdir(self.workdir)
```

Comment:
As we now, give the cwd to each needed bash command. Lets do not use `os.chdir()` here.

Reply:
done